File: IOS_swift/WitSDK/WitSensorApi/Modular/Ble5/IBwt901bleRecordObserver.swift
//
//  bwt901ble数据记录观察者
//
//  Created by 赵文 on 2022/9/3.
//

import Foundation

public protocol IBwt901bleRecordObserver {
    
    // 记录数据事件
    func onRecord(_ bwt901ble:Bwt901ble)
    
}

File: Android_Java/Old_Example/wit-example-ble5/app/src/main/java/com/wit/example/ble5/interfaces/IBwt901bleRecordObserver.java
package com.wit.example.ble5.interfaces;


import com.wit.example.ble5.Bwt901ble;

/**
 * 记录数据通知
 *
 * @author huangyajun
 * @date 2022/4/26 11:27
 */
public interface IBwt901bleRecordObserver {

    /**
     * 接收通知
     *
     * @author huangyajun
     * @date 2022/4/26 13:42
     */
    void onRecord(Bwt901ble bwt901ble);

}
